DQM/DTMonitorModule/src/DTDCSByLumiTask.cc

```diff
@@ -121,6 +121,7 @@ void DTDCSByLumiTask::dqmEndLuminosityBlock(const edm::LuminosityBlock& lumiSeg,
     // wires list
 
     if (DTHVRecordFound) {
+      [[clang::suppress]]
```

Comment:
@iarspider , may be instead of suppressing it if you change the condition above to `if (dtHVStatus) {` then clang-analyzer might be happy